src/modules/site-v2/base/views/strains.py

```diff
@@ -225,7 +225,7 @@ def order_page_post():
           return redirect(url_for('request_strains.order_page_index'))
         
         if form.shipping_service.data == 'Flat Rate Shipping':
-          cartItems.append({'name': 'Flat Rate Shipping'})
+          cartItems.append({'name': 'Flat Rate Shipping', 'species': ''})

```

Comment:
Can/should this use `Cart.add_item`, instead of appending directly?

This is a bit subjective, but I think it's usually safer to ask the `Cart` object for information, rather than using the `items` prop directly.  That way, if we need to do something fancy behind-the-scenes with `Cart`, we don't have to change any of this code that uses it.

--

Not required for this PR, but a few ideas I'll mention here anyways:

- If you want to learn more about Python "dunder" methods (dumb name I know), you could try implementing `Cart.__len__` and using `len(users_cart)` a few lines above here (instead of calling `len` on `users_cart['items']`.
- You might also consider making a `Cart.is_empty()` method, in the interest of hiding implementation details inside the `Cart` class.

You could try looking into those now, or put them off for later if you have more urgent work :)